packages/manager/src/features/CloudPulse/Alerts/AlertsListing/AlertListing.tsx

```diff
@@ -33,6 +36,11 @@ export const AlertListing = () => {
     isLoading: serviceTypesLoading,
   } = useCloudPulseServiceTypes(true);
   const topRef = React.useRef<HTMLButtonElement>(null);
+
+  const canCreateAlerts =
```

Comment:
```suggestion
  const isAlertLimitReached =
```

this name or similar to it is much better for the condition that is checked here